README.md

```diff
@@ -101,10 +101,16 @@ browsers.
 * Provides a workaround for [this bug](https://github.com/webrtc/adapter/issues/714), where webrtc-adapter's shimmed
   `addTrack` method does not return the `RTCRtpSender` associated with the added track.
 
+#### Edge
+* Uses `otalk/rtcpeerconnection-shim` for WebRTC API implementation
```

Comment:
@syerrapragada Nitpick: Instead of ``, can you convert this to a link to the upstream github project? Also, you can re-write this sentence as:
```
* Uses [otalk/rtcpeerconnection-shim](github_url) to implement WebRTC APIs on top of ORTC.
```